docs/api/utility-process.md

```diff
@@ -0,0 +1,128 @@
+# UtilityProcess
```

Comment:
When the API is fully finalized, we should also update our explainer docs (e.g. Process Model, Tutorial series) to reflect the existence of this new process type.